### DISCUSSION TITLE
[Suggestion] New CvC Endpoints

### DISCUSSION THREAD

**Discussion by baseballaholic** [HypixelDev/PublicAPI#470] [Ideas]
Hey!

I was wondering if it could be possible for new cvc endpoints to be added into the API. This is practically my first time using github so I don't know if this is something that is normally done so my apologies if this is unusual. The endpoints I hope to be added are for tracking if a gun has the ammo clip upgrade purchased. Currently the API stores all upgrade information about guns except for that single upgrade (on each gun).

**Comment by Picsou993** [comment]
Hia!

This information is already available into the API.
These Ammo Clips Upgrades are a One-Time Purchases as you said and for some One-Time, they are using a boolean, some others are using an int = 1, and some others which is the cases for these ones are stored within the packages.

So you can find if a player has purchased the upgrade for a specific gun at `player.stats.MCGO.packages` and chech if `GunName_clip` exists or not. If it does then the player has purchased it.

**Reply by baseballaholic** [reply]
Guess I must have missed it, I always closed packages when I was looking at the API because I thought it was just the sprays and callouts, thanks Picsou!
